src/components/Mission/Mission .js

```diff
@@ -1,48 +1,79 @@
-import React from 'react';
-import { Table } from 'react-bootstrap';
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { Table, Container } from 'react-bootstrap';
+import { getMissionsData, JoinMission } from '../../redux/mission/mission';
 
-const MyMission = () => (
-  <>
-    <Table striped bordered hover size="lg" className="container my-5">
-      <thead>
-        <tr>
-          <th>Mission</th>
-          <th>Description</th>
-          <th>Status</th>
-        </tr>
-      </thead>
-      <tbody>
-        <tr>
-          <td>Mark</td>
-          <td>Otto</td>
-          <div className="btns">
-            <tr>
-              <button type="button" className="btn btn-secondary ms-3">
-                NOT A MEMBER
-              </button>
-              <button type="button" className="btn btn-outline-dark ms-3">
-                Join Mission
-              </button>
-            </tr>
-          </div>
-        </tr>
-        <tr>
-          <td>Jacob</td>
-          <td>Thornton</td>
-          <div className="btns">
-            <tr>
-              <button type="button" className="btn btn-info ms-3">
-                NOT A MEMBER
-              </button>
-              <button type="button" className="btn btn-outline-danger ms-3">
-                Join Mission
-              </button>
+function Mission() {
```

Comment:
Hi, you've done a great job so far, just a few changes to make sure everything works well 
Change the name of the function from the previous one, so that it could fit with the nav link router.

Instead of `function Mission()` change that name for `function MyMission()`